packages/webcert/src/feature/careProvider/CareProviderModal.test.tsx

```diff
@@ -64,6 +74,8 @@ describe('Care provider modal', () => {
   it('should close modal when unit is chosen', async () => {
```

Comment:
Here it seems like you are referring to backend logic in your frontend test. The test itself might not be wrong but from my point of view it doesn't seem like the text explains what the test is actually testing. Isn't what you actually want to test if the user chooses a unit then certain api-calls are made (api-call to choose unit, and also maybe to update user and to update the user resource links)? And another test which checks if these api-calls results in the resource link for choose unit to be missing then the modal closes?

Reply:
The name of the test is correct because I'm only testing the user interaction. The other tests that you talk about are among the user middleware tests.